xml/operations-maintenance-controller-pit_swiftrings_recovery.xml

```diff
@@ -6,110 +6,163 @@
- <title>Point-in-Time Swift Rings Recovery</title>
+ <title>Point-in-Time &swift; Rings Recovery</title>
  <para>
-  In this situation, everything is still running (&lcm;, control
-  plane nodes, and compute nodes) but you want to restore your Swift rings to a
-  previous state.
- </para>
- <para>
-  In this scenario, everything is still running (&lcm;, control
-  plane nodes, and compute nodes) but you want to restore your Swift rings to a
-  previous state.
+  In this situation, everything is still running (&lcm;, control plane nodes,
+  and compute nodes) but you want to restore your &swift; rings to a previous
+  state.
  </para>
  <note>
   <para>
-   Freezer backs up and restores Swift rings only, not Swift data.
+   Freezer backs up and restores &swift; rings only, not &swift; data.
   </para>
  </note>
- <section>
-  <title>Restore from a Swift backup</title>
-  <orderedlist>
-   <listitem>
+ <section xml:id="restore-swift-bu">
+  <title>Restore from a &swift; backup</title>
+  <procedure>
+   <step xml:id="swift-nodes">
     <para>
-     Log in to the first Swift Proxy (<literal>SWF-PRX[0]</literal>) node.
+     Log in to the first &swift; Proxy (<literal>SWF-PRX[0]</literal>) node.
     </para>
-   </listitem>
-   <listitem>
+    <para>
+     To find the first &swift; Proxy node:
+    </para>
+    <substeps>
+     <step>
+      <para>
+       On the &lcm;
+      </para>
+<screen>cd  ~/scratch/ansible/next/ardana/ansible
+    ansible-playbook -i hosts/verb_hosts swift-status.yml \
+    --limit SWF-PRX[0]</screen>
+      <para>
+       At the end of the output, you will see something like the following
+       example:
+      </para>
+<screen>
+<?dbsuse-fo font-size="0.65em"?>
+...
+Jun 18 20:01:49 ardana-qe102-cp1-c1-m1-mgmt swiftlm-uptime-mon[3985]: 'uptime-mon - INFO : Metric:keystone-get-token:max-latency: 0.679254770279 (at 1529352109.66)'
+Jun 18 20:01:49 ardana-qe102-cp1-c1-m1-mgmt swiftlm-uptime-mon[3985]: 'uptime-mon - INFO : Metric:keystone-get-token:avg-latency: 0.679254770279 (at 1529352109.66)'
+
+PLAY RECAP ********************************************************************
+ardana-qe102-cp1-c1-m1 : ok=12 changed=0 unreachable=0 failed=0```</screen>
+     </step>
+     <step>
+      <para>
+       Find the first node name and its IP address. For example:
+      </para>
+<screen>cat /etc/hosts | grep ardana-qe102-cp1-c1-m1</screen>
+     </step>
+    </substeps>
+   </step>
+   <step>
     <para>
      Become root:
     </para>
 <screen>sudo su</screen>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      Source the backup environment file:
     </para>
-<screen>source /home/stack/backup.osrc</screen>
-   </listitem>
-   <listitem>
+<screen>source /var/lib/ardana/backup.osrc</screen>
+   </step>
+   <step>
+    <para>
+     Find the client id.
+    </para>
+<screen>freezer client-list
+<?dbsuse-fo font-size="0.65em"?>
```

Comment:
Prompt depends on whether one has to become root (prompt.root) or not (prompt.ardana)

          <screen><?dbsuse-fo font-size="0.65em"?>&prompt.????freezer client-list